packages/contractkit/src/utils/rpc-caller.ts

```diff
@@ -110,3 +128,13 @@ export class DefaultRpcCaller implements RpcCaller {
     }
   }
 }
+
+export interface TelemetryHandler {
+  handleEvent: (path: string[], measurements: { duration: number }, metadata: any) => void
+}
+
+export class DebugTelemetryHandler implements TelemetryHandler {
+  handleEvent(path: string[], measurements: { duration: number }, metadata: any) {
+    debugRpcTelemetry('Path: %o, Duration: %s, metadata: %o', path, measurements.duration, metadata)
```

Comment:
where do you want to use this? You're using the debug handler, which is not a great fit if this is something you want always "on" for some cases

Reply:
Yeah I think the debug handler is a misnomer as we will probably always want to just add it to `debug`? The intention was to use this in the mobile wallet to measure eth_calls